docs/resource_sets/resource_sets_sources/basic_example.cf

```diff
@@ -0,0 +1,9 @@
+
+entity Res extends std::Resource:
```

Comment:
I would not use the name `Res` here. It would be better to use a real world thing that users understand. I think `Router` would be a good candidate. That way it also becomes easier to explain the example. You can say for example that resource sets allow a user to only update a subset of the Routers.